### ISSUE TITLE
lacking rom for download and no screenshots

### ISSUE THREAD

**Issue by ghost** [pinobatch/nesbgeditor#1]
this repo could have done with having some screenshots and a rom for immediate download.

**Comment by pinobatch** [comment]
I agree that a release is a good idea. What screens in the editor would be worthwhile to show as screenshots?

**Comment by ghost** [comment]
unsure, not tried yet. but when i do I will report back. thanks for swift reply m8
